src/main/java/org/fenixedu/academic/json/adapters/DepartmentJsonAdapter.java

```diff
@@ -16,7 +16,7 @@
  * You should have received a copy of the GNU Lesser General Public License
  * along with FenixEdu Academic.  If not, see <http://www.gnu.org/licenses/>.
  */
-package org.fenixedu.academic.ui.spring.controller.manager;
+package org.fenixedu.academic.json.adapters;
```

Comment:
While this is the package the adapter should be defined on, changing it would be an API break.